docker/runtime/Dockerfile

```diff
@@ -51,6 +51,7 @@ RUN set -x && apt-get update -qq \
     libgit2-dev \
     libssl1.1 \
     libgvc6 \
+    libldap-2.4-2 \
```

Comment:
You add the LDAP dependency, but CodeCompass is built without the ldap auth plugin.